### PR TITLE
feat: :sparkles: updates schema to allow multiple maven artifacts

### PR DESCRIPTION
**Pull request by dsoffiantini**
## Description

Updates for Android 2.0 support, allows specifying a core version and multiple artifacts

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Do these changes need to be coordinated with changes in other repositories?

Needs to be released before Android 2.0 updates

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by dgrossen — APPROVED** [review]

**Comment by dgrossen** [comment]
@jeffreywalter when should we plan on this to be merged in and deployed?

File: schemas/extension-package-mobile.json [hunk 1]
```diff
@@ -117,13 +113,33 @@
...
+        {
+          "type": "object",
+          "properties": {
+            "className": {
+              "$ref": "#/definitions/javaClassName"
+            },
+            "artifact": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/mavenFQArtifactName"
```
Comment (dgrossen) [inline schemas/extension-package-mobile.json:133]: Shouldn't it be className that is an array? I don't think we want artifacts as an array

**Review by dgrossen — CHANGES_REQUESTED** [review]
Please look at the new Maven repo object

**Comment by dgrossen** [comment]
Just change `artifact` to `artifacts` when it's an array and then I think it's good

**Review by dgrossen — APPROVED** [review]

**Review by brenthosie — APPROVED** [review]

**Pull request merged**